env/cloud/alarms/terragrunt.hcl

```diff
@@ -71,16 +71,16 @@ dependency "lambdas" {
   mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
   mock_outputs_merge_strategy_with_state  = "shallow"
   mock_outputs = {
-    lambda_reliability_log_group_name                = "/aws/lambda/Reliability"
-    lambda_submission_log_group_name                 = "/aws/lambda/Submission"
-    lambda_response_archiver_log_group_name          = "/aws/lambda/Response_Archiver"
-    lambda_dlq_consumer_log_group_name               = "/aws/lambda/DeadLetterQueueConsumer"
-    lambda_template_archiver_log_group_name          = "/aws/lambda/Archive_Form_Templates"
-    lambda_audit_log_group_name                      = "/aws/lambda/AuditLogs"
-    lambda_nagware_log_group_name                    = "/aws/lambda/Nagware"
-    lambda_vault_data_integrity_check_log_group_name = "/aws/lambda/Vault_Data_Integrity_Check"
-    lambda_vault_data_integrity_check_function_name  = "Vault_Data_Integrity_Check"
-    lambda_audit_logs_archiver_group_name            = "/aws/lambda/Audit_Logs_Archiver"
+    lambda_audit_logs_log_group_name               = "/aws/lambda/audit-logs"
```

Comment:
Super minor comment, but now that you're using the legacy CloudWatch log group names, you could also revert these mock values to the old log group names as well.

But it's really just a preference thing since either will work!

Reply:
True! It was on the todo list I had in my head and of course I forgot about it 😆

Reply:
Hah ever since having kids my todo list is continually forgotten and overwritten